web-calculator.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <div class="calcBody">
+        <h1>Calculator</h1>
+    <input type="number" id="num1">
```

Comment:
intentation issue. everything inside div(.calcbody) should be indented a bit more.